### PR TITLE
PLATUI-848: Uplift SBT to 1.4

### PR DESCRIPTION
**Pull request by matthewmascord**

### CODE REVIEW

File: project/project/PlayCrossCompilation.scala [hunk 1]
```diff
@@ -3,10 +3,7 @@ import uk.gov.hmrc.playcrosscompilation.PlayVersion._
 
 object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play27) {
   val playRevision: String = PlayCrossCompilation.playVersion match {
-    case Play27 => "2.7.5"
-    case Play26 => "2.6.23"
-    case Play25 => throw new IllegalArgumentException("Play 2.5 is no longer supported by this library")
+    case Play26 => "2.6.20"
+    case Play27 => "2.7.4"
```
Comment (JoPintoPaul) [inline project/project/PlayCrossCompilation.scala]: Has the `PlayCrossCompilation.playVersion` enum had Play 2.5 removed?

**Comment by JoPintoPaul** [comment]
@matthewmascord I see that the specific folder for play26 has been renamed. Assuming the library still builds as expected?

**Comment by matthewmascord** [comment]
> @matthewmascord I see that the specific folder for play26 has been renamed. Assuming the library still builds as expected?

Yep I also ran into an issue because the cross compilation library because you can't re-use the play26 directory for play27 builds.

File: project/project/PlayCrossCompilation.scala (hunk 1 above)
Comment (matthewmascord) [reply project/project/PlayCrossCompilation.scala]: Added back just in case someone tries to compile for Play 2.5.

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**